fendermint/vm/interpreter/src/fvm/mod.rs

```diff
@@ -91,7 +87,7 @@ where
     pub fn new(
         client: C,
         validator_ctx: Option<ValidatorContext<C>>,
-        contracts_dir: PathBuf,
+        _contracts_dir: PathBuf,
```

Comment:
Any reason we don't remove this arg?

Reply:
Good catch, I missed it. Was using it as a placeholder so that not to break so many code at once. Will remove it.